src/TrueLayer/Payments/Model/PaymentUserRequest.cs

```diff
@@ -11,11 +15,21 @@ public record PaymentUserRequest
         /// <param name="name">The user's name.</param>
         /// <param name="email">The user's email address.</param>
         /// <param name="phone">The user's phone number.</param>
-        public PaymentUserRequest(string? name = null, string? email = null, string? phone = null)
+        /// <param name="dateOfBirth">The user's date of birth</param>
+        /// <param name="address">The user's physical address</param>
+        public PaymentUserRequest(
+            string? name = null,
+            string? email = null,
+            string? phone = null,
+            DateTime? dateOfBirth = null,
+            Address? address = null)
```

Comment:
If the props are mandatory, shouldn't we make them non-nullable?

Reply:
`Address` is not mandatory
![image](https://user-images.githubusercontent.com/85947184/220641673-27b99ddb-9423-4c9a-9746-82dc047067a7.png)
Some of its properties are mandatory only if the `address` is specified

Reply:
That is true but the SDK has to stay stick to what the doc of our API says. We can't add more checks than what the API actually does. 🤷🏻‍♂️

Reply:
✅ Let's just make sure we raise this to payments-api so that they can sort it out.

Reply:
I have asked the documentation and payments team about it. Will merge the PR if I get the go from them.